testsuite/tests/kuadrant/authorino/performance/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""
+Conftest for performance tests
+"""
+import os
+from pathlib import Path
+
+import pytest
+from hyperfoil import HyperfoilClient
+
+from testsuite.perf_utils import HyperfoilUtils
+from testsuite.httpx.auth import HttpxOidcClientAuth
+
+
+@pytest.fixture(scope='session')
+def hyperfoil_client(testconfig):
+    """Hyperfoil client"""
+    client = HyperfoilClient(testconfig['hyperfoil']['url'])
+    return client
+
+
+@pytest.fixture(scope='module')
+def hyperfoil_utils(hyperfoil_client, template, request):
+    """Init of hyperfoil utils"""
+    utils = HyperfoilUtils(hyperfoil_client, template)
+    request.addfinalizer(utils.finalizer)
+    return utils
+
+
+@pytest.fixture(scope='session')
+def root_path():
+    """Root path for performance tests"""
+    return Path(os.path.realpath(__file__)).parent
+
+
+@pytest.fixture(scope="module")
+def rhsso_authorization(authorization, rhsso):
```

Comment:
Should be set up by default already, if you want to force RHSSO, only thing that's needed is to override `oidc_provider` fixture

Reply:
Removed